iron-focusables-helper.html

```diff
@@ -127,18 +127,17 @@
         //  </div>
         // TODO(valdrin) support ShadowDOM v1 when upgrading to Polymer v2.0.
```

Comment:
So turns out this implementation is buggy also in SD v0 when there are elements with tabindex > 0, so most likely I'll either leave this as is for now. More details https://github.com/PolymerElements/iron-overlay-behavior/pull/241